test-data/unit/semanal-errors.test

```diff
@@ -373,11 +373,6 @@ main:1: error: can't assign to literal
 [out]
 main:1: error: can't assign to literal
 
-[case testInvalidLvalues5]
```

Comment:
Shouldn't this still be an error saying `1 is not iterable` or similar?

Also, I would like to see a new test that does things like `() = []` or `a, () = list()` and asserts that there are no errors.